### PR TITLE
Move add button widget to the widgets tab

### LINKED ISSUE #6811: Make "Add widgets" button only appear in the widgets tab
Right now it's confusing to have it in all the tabs.

### PR DESCRIPTION
**Pull request by javierarce**
Closes #6811

### CODE REVIEW

**Comment by javierarce** [comment]
CR: @xavijam

File: lib/assets/javascripts/cartodb3/editor/editor-view.js [hunk 1]
```diff
@@ -29,6 +27,7 @@ module.exports = cdb.core.View.extend({
     var stackViewCollection = new Backbone.Collection([{
       createStackView: function (stackLayoutModel, opts) {
         return new EditorWidgetsView({
+          modals: self._modals,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/editor-view.js]: What does this variable `modals` do?

Comment (javierarce) [inline lib/assets/javascripts/cartodb3/editor/editor-view.js]: It's required to create modals: https://github.com/CartoDB/cartodb/pull/6824/files#diff-fb316265868d569d35e3b2ebd4888dfaR41

Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/editor-view.js]: Aaaajam! Ok!

**Comment by xavijam** [comment]
Go baby go! :)

**Pull request merged**